projects/plugins/jetpack/modules/google-fonts.php

```diff
@@ -115,6 +132,80 @@ function jetpack_add_google_fonts_provider() {
 }
 add_action( 'after_setup_theme', 'jetpack_add_google_fonts_provider' );
 
+/**
+ * Updates font names by filtering the data provided by the theme for global styles & settings.
+ *
+ * `wp_register_webfonts()` does not currently support giving a separate "name"
+ * from "font-family", even if theme.json schema itself does support it.
+ * See: https://github.com/WordPress/gutenberg/issues/46398
+ *
+ * Once support for "name" is added, below functionality can be refactored
+ * to provide the name directly within `wp_register_webfonts()`.
+ *
+ * @param WP_Theme_JSON_Data_Gutenberg $theme_json Class to access and update the underlying data.
+ * @return WP_Theme_JSON_Data_Gutenberg Class with updated Global Styles settings.
+ */
+function jetpack_rename_google_font_names( $theme_json ) {
+
+	// Rename only in the editor and API requests
+	if ( is_admin() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
+		$raw_data = $theme_json->get_data();
+
+		// Skip if fontFamilies are not defined in the variation.
+		if (
+			empty( $raw_data['settings'] ) ||
+			empty( $raw_data['settings']['typography'] ) ||
+			empty( $raw_data['settings']['typography']['fontFamilies'] ) ||
+			empty( $raw_data['settings']['typography']['fontFamilies']['theme'] )
```

Comment:
You should just need this one empty check, if any of the other keys are empty this one will be, too